extra/jaeger.yml

```diff
@@ -0,0 +1,5 @@
+apiVersion: jaegertracing.io/v1
```

Comment:
Instead of having the tracing files under `extra` can we make the directory called `tracing` ?